plugin-server/src/worker/vm/upgrades/utils/fetchEventsForInterval.ts

```diff
@@ -69,7 +69,7 @@ const convertClickhouseEventToPluginEvent = (event: RawClickHouseEvent): Histori
         event: clickhouseEvent.event || '',
         ip: clickhouseEvent.properties['$ip'] || '',
         site_url: '',
-        sent_at: clickhouseEvent.created_at.toISO(),
+        sent_at: clickhouseEvent.timestamp.toISO(),
```

Comment:
https://posthog.slack.com/archives/C0374DA782U/p1670877989482779?thread_ts=1670877439.683469&cid=C0374DA782U

Reply:
We want to ensure that the timestamp of events migrated via e.g. Replicator (or any other exporter) is the same as the event in the ClickHouse db, otherwise any analysis based on these exported events will be inaccurate.

Reply:
For instance, we want to ensure that calls to `/e/` by replicator include the `sent_at`, as this is the only way to override the ClickHouse Event timestamp column: https://github.com/PostHog/posthog-plugin-replicator/pull/10/files